flutter_bird_app/lib/view/authentication_popup.dart

```diff
@@ -184,11 +184,13 @@ class _AuthenticationPopupState extends State<AuthenticationPopup> {
     );
   }
 
-  _buildQRView(String data) => QrImage(
-        data: data,
-        version: QrVersions.auto,
-        size: 200.0,
-      );
+  _buildQRView(String data) {
+    return QrImageView(
```

Comment:
Is this a change caused by an update to qr_flutter?
Also, does this affect any functionality?

Reply:
Yes, this is the name of qr_flutter v4.0.1 and later, which was updated at the same time. There is no functional difference.